cloudformation.yaml

```diff
@@ -203,8 +203,6 @@ Resources:
       Name: !Sub ${Stack}-${Stage}-${App}
       Port: 9000
       Protocol: HTTP
-      VpcId:
-        Ref: VpcId
```

Comment:
this is the only change. Here `VpcId` appeared twice (by mistake), which now produces an error:
`Map keys must be unique; "VpcId" is repeated`

Reply:
good to fix this!